bindata/v4.1.0/kube-apiserver/localhost-recovery-token.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: localhost-recovery-client-token
+  annotations:
+    kubernetes.io/service-account.name: localhost-recovery-client
+type: kubernetes.io/service-account-token
```

Comment:
any reason this is not a simple CVO manifest?

Reply:
It is operand's config, not operator's - I think these go here

If we ever need to change it or extend it with any additional data, we should be controlling that resource, not CVO. If we want to migrate away from it, we should be controlling it.

Reply:
Don't we have operand config as manifest too in many places? I don't think complexity for future potential needs makes sense here.